src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -7423,20 +7423,20 @@ void func_8083EA94(Player* this, PlayState* play) {
     Player_AnimPlayOnce(play, this, GET_PLAYER_ANIM(PLAYER_ANIMGROUP_throw, this->modelAnimType));
 }
 
-s32 func_8083EAF0(Player* this, Actor* actor) {
-    if ((actor != NULL) && !(actor->flags & ACTOR_FLAG_23) &&
+s32 Player_CanDropCarriedActor(Player* this, Actor* actor) {
+    if ((actor != NULL) && !(actor->flags & ACTOR_FLAG_THROW_ONLY) &&
         ((this->speedXZ < 1.1f) || (actor->id == ACTOR_EN_BOM_CHU))) {
-        return 0;
+        return false;
     }
 
-    return 1;
+    return true;
```

Comment:
This function name seems backwards. If you can drop the carried actor, then `false` is returned, and if you can throw, then `true` is returned. You can see it clearly here as well, thanks to the `!` in there
```c
if (!Player_CanDropCarriedActor(this, heldActor)) {
    doAction = DO_ACTION_DROP;
} else {
    doAction = DO_ACTION_THROW;
}
```

Reply:
oh yeah, whoops. fixed